spring-cloud-stream-samples/pom.xml

```diff
@@ -4,7 +4,6 @@
 	
 	<artifactId>spring-cloud-stream-samples</artifactId>
 	<packaging>pom</packaging>
-	<url>http://projects.spring.io/spring-xd/</url>
```

Comment:
We can use the correct URL `http://cloud.spring.io/spring-cloud-stream/` instead.